docs/C_STYLE_GUIDE.md

```diff
@@ -0,0 +1,112 @@
+# C Style Guide
+
+This guide describes the style intended for use with the C track exercise 
+example and test files.
```

Comment:
A minor issue: Exercism-wide it was decided to not add manual breaks in lines in Markdown documents. In order to keep things fairly readable (and writable), each sentence should be on its own line. This makes it so that lines don't get forever long in editors that don't soft wrap while also making it so that minor updates to a Markdown document don't require lines to be manually re-wrapped.

This is a fairly recent thing and I'm personally okay with doing no manual wrapping, but if we are going to wrap, let's do it in the recommended way.

Reply:
Ah, good to know, I'll adjust to suit